proprietary/tokens/src/brand/ams/typography.tokens.json

```diff
@@ -1,12 +1,29 @@
 {
   "ams": {
-    "text": {
+    "typography": {
       "font-family": { "value": "'Amsterdam Sans', Arial, sans-serif" },
-      "font-weight": {
-        "normal": { "value": 400 },
-        "bold": { "value": 800 }
+      "body-text": {
+        "font-size": { "value": "clamp(1.125rem, calc(1.0313rem + 0.4688vw), 1.5rem)" },
```

Comment:
If we want to communicate that the 2 lists partially overlap, we could use a reference here. Something like `{ams.typography.heading.5.font-size}`. That would also prevent inconsistencies.

Reply:
That’s because we made our ‘layer above brand’ invisible. You could say we lost the same kind of information in our colour tokens, where nothing explicitly indicates that our invalid red and error red stem from the same colour code.

So, relating a body text size to a heading size is incorrect – they have the same ancestor. Even more so because the only reason for a ‘level 5 heading’ font size to exist is to allow a Heading to match the font size of a paragraph.